pkg/fleet/installer/installer.go

```diff
@@ -945,3 +962,32 @@ func ensureRepositoriesExist() error {
 
 	return nil
 }
+
+// mergeConfigs merges multiple config files into a single map of config files.
```

Comment:
q: rawConfigs contains Remote Config configs, and we output what we'd write on disk, right?